HouseRules_Essentials/README.md

```diff
@@ -27,6 +27,8 @@ HouseRules API.
 Rulesets may also be configured as JSON files and stored within the game directory `<GAME_DIR>/UserData/HouseRules/<rulesetname>.json`
 An example [LuckyDip Ruleset](../docs/LuckyDip.json) which uses many differnt rules for rapid gameplay is provided as a guide to help you get started.
 
+The [Settings Reference](../docs/LuckyDip.json) contains lists of all different BehaviourIDs, AbilityKeys and other data types used by the Rules.
```

Comment:
Small path typo.

Reply:
Good catch - thank you. I've force pushed a fixed commit.